docs/update_kops.md

```diff
@@ -2,12 +2,19 @@
 
 Update the latest source code from kubernetes/kops
 
-```
+```bash
```

Comment:
Not on you, but I do not like this document much.  We have had issues with a lot of people installing from source in the past.  Wondering if we should refer them just include the base install instructions here that are in the README.

Reply:
Or just add notes that we just replace the binary to update kops in the install doc, and link to that document in this doc.

Reply:
Okay that makes sense. I'll rework this pending the above comments ^.  We probably should just refer them to docs/development for the `git pull && make` version.